g2p_odk_importer_documents/models/odk_import.py

```diff
@@ -52,6 +52,8 @@ def process_records_handle_media_import(self, mapped_json, member):
             attachm = None
             if filename:
                 attachm = self.odk_config.download_attachment(instance_id, filename)
+            else:
+                continue
```

Comment:
This is good for now, but needs discussion. For example, if an attachment is not uploaded against a form field, can we create a placeholder document object to somehow represent that no file was uploaded against this document?